### ISSUE TITLE
Add a scroll down action when double tap on the back of the console

### ISSUE THREAD

**Issue by Lyr** [Creckeryop/NOBORU#25]
It could be very usefull to add new action when double tap/click on the back of the console (or on the front but this one is yet provide for zoom) to pull down the screen. I mean not the next page but the next part of the page the not be visible. With this action, it will be more quickly to slide the page without the analogik pad.

**Comment by Creckeryop** [comment]
screen slide isn't working?

**Comment by Creckeryop** [comment]
i just don't know how to predict where you are reading your manga up or down or left or right, should i write neural network for this?

**Comment by Lyr** [comment]
Good point  for the prediction, and yes slide is working, but it will be usefull to be able to make this action to slide a full screen with just one tap on back (or on front, it could be configure).
For the up/down left/righ, it could be good to fix region on the back for each of them. I mean to keep it simple, left region for slide to left top to top etc...

**Comment by Creckeryop** [comment]
are you sure that this will be simple for user?

**Comment by Creckeryop** [comment]
there will be a lot of missclicks, and this just doesn't comfortable to use even in vertical mode

**Comment by Creckeryop** [comment]
where i can see that feature implemented?

**Comment by Lyr** [comment]
This feature exists on some open source reader in android. I'll try to search but I don't found the one who do this. I think there are 2 point on this enchancement :

- The feature (abilitie to scroll the full "screen window" down/up/left/right) with just one action of user

- The way to catch this action (I've in mind the back of vita but it could be difficult at the first, I guess a map of other button it's ok for the begining)

**Comment by Creckeryop** [comment]
why you can't scroll with analog stick? and what's mean full screen window

**Comment by Creckeryop** [comment]
only start button is free for that action, don't think this will be nice

**Comment by Lyr** [comment]
Yes I'm agree the start button is not correct for this action. I can scroll with analog stick, but 2 things about this :
- This action is to scroll not to go to the next visualization (not the next page, but currently you scroll full size of the visu and my point is to put directly to the next visualization)(visualization is the part of the page displaying on your screen)
- The analog stick becomes difficult to handle during long reading sessions

**Comment by Creckeryop** [comment]
Interesting idea, hope i will implement it some time
